### ISSUE TITLE
My hunts - on mobile interface

### ISSUE THREAD

**Issue by sydneyitguy** [Steemhunt/web#316]
![image](https://user-images.githubusercontent.com/1332279/56208343-e8904380-608b-11e9-8846-e151fde2bc90.png)

**Comment by Chuuuckie** [comment]
Yes, also add NUMBERS in MOBILE. I cant see numbers like TOP 1,2,3,4,5 D:

**Comment by sydneyitguy** [comment]
@Chuuuckie We couldn't find a good solution to put numbers on mobile interface with a good design. We will think about it again when we do design-renewal on Steemhunt. Please create a separate ticket for the issue.

**Comment by wongandy** [comment]
hi! any updates regarding this ticket?

**Comment by sydneyitguy** [comment]
@wongandy We have some other works in higher priority at the moment (reviewhunt).
We're thinking of a big update on Steemhunt UI in few months from now.
